Week04/lesson08/homework/temp_converter/js/indexbonus3.js

```diff
@@ -0,0 +1,57 @@
+function conversion() {
+
+  var c = document.querySelector("#c").value;
+  var f = document.querySelector("#f").value;
+
+  if ((f === null || f === "") && (c === null || c === "")) {
+      document.querySelector("#alert").innerHTML = "Please fill one of the fields above.";
+  }
+
+  else if (f === null || f === "") {
+    f = c * 9 / 5 + 32;
+    document.querySelector("#f").value = Math.round(f*10)/10+"°F";
+    c = c+"°C";
+  }
+
+  else if (c === null || c === "") {
+    c = (f - 32) * 5 / 9;
+    document.querySelector("#c").value = Math.round(c*10)/10+"°C";
```

Comment:
I did think about it but this means that if I try to convert 0°C it will systematically give me 32.0°F if I use toFixed(1) or 32.00°F if I use toFixed(2) rather than a nice and clean 32°F when using Math.round.